### ISSUE TITLE
Make ALL_IN_SCHEMA for tables affect views

### ISSUE THREAD

**Issue by ToBeReplaced** [ansible/ansible-modules-core#833]
##### Issue Type:

Bug Report
##### Ansible Version:

ansible 1.8.2
##### Environment:

N/A
##### Summary:

ALL_IN_SCHEMA on tables does not include views. It should, as per http://www.postgresql.org/docs/9.3/static/sql-grant.html :

```
There is also an option to grant privileges on all objects of the same type within one or more 
schemas. This functionality is currently supported only for tables, sequences, and functions (but
note that ALL TABLES is considered to include views and foreign tables).
```
##### Steps To Reproduce:

On a database with a role that has had all privileges revoked:

```
- name: Give SELECT privilege to site on all tables and views
  postgresql_privs:
    db=some_database
    privs=SELECT
    objs=ALL_IN_SCHEMA
    role=some_role
```
##### Expected Results:

`some_role` should be able to select on a view in the public schema of `some_database`.
##### Actual Results:

`some_role` cannot select on a view in the public schema of `some_database`.
##### Details:

ALL TABLES is considered to include views, so we must check for reltypes
'r' and 'v', not just 'r'. A pull request was submitted last July before the module split. See discussion here:

https://github.com/ansible/ansible/pull/8178

**Comment by ToBeReplaced** [comment]
Note that this bug only exists because the backwards-compatibility code is incorrect. If one used the modern >9.0 "ALL TABLES IN SCHEMA" language, this problem would not occur.

**Comment by hydrapolic** [comment]
Please apply this PR. It is not possible to set up permissions on views with Ansible when using ALL_IN_SCHEMA.

**Comment by gregdek** [comment]
The PR was applied against the old repository structure. @ToBeReplaced or @hydrapolic please feel free to re-file the original PR against ansible/ansible-modules-core and we'll put it into our review process as soon as we can.

**Comment by ToBeReplaced** [comment]
PR at #1621 

I hadn't recreated the PR before because it didn't seem like it was going to get merged (due to lack of review in https://github.com/ansible/ansible/pull/8178). Thank you @hydrapolic for adding your experience so that this will be fixed.

**Comment by hydrapolic** [comment]
I've tested the solution by @ToBeReplaced and it worked ok. Now setting up permissions with ALL_IN_SCHEMA worked for tables and views, great!

**Comment by bcoca** [comment]
closed by merging #1621

**Comment by hydrapolic** [comment]
Thanks @bcoca and @ToBeReplaced.
